types/express-serve-static-core/index.d.ts

```diff
@@ -1252,3 +1249,24 @@ export interface Express extends Application {
     request: Request;
     response: Response;
 }
+
+export interface Layer {
+    handle: () => void;
+    name: string | '<anonymous>';
+    params: any;
+    path: any;
+    regexp: RegExp;
+    route?: Route;
+}
+
+export interface Route {
```

Comment:
How is Route different from IRoute?

Reply:
If its better for you that the methodes object is in the IRoute interface i could propably change that but i dont think that this is accurat to the structure of the Layers in the Stack